docs/Installation-Guide.md

```
@@ -67,7 +71,30 @@ npm run dev      # development
 npm run build    # production
 ```
 
-The editor mounts on every page that contains a `[data-ap-visual-editor]` element.
+The editor mounts on every page that contains a `[data-ap-visual-editor]` element. Use this when the host already owns a Vite pipeline and you want editor sources to hot-reload during development.
+
+### Pattern B — serve pre-built bundles from `vendor/`
+
+Since v1.5.2 the Composer tarball ships pre-built editor bundles under `vendor/artisanpack-ui/visual-editor/dist/editor/`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)